### PR TITLE
Tunes for developers docs

### LINKED ISSUE #528: Doc on creating the notification tunes (sounds)
Tunes are played on startup, error, and other notification. We need a doc explaining how this is done. It should cover:

- What are tunes
- What the default tunes (and are there non-default tunes?)
- Where are they defined? https://github.com/PX4/Firmware/tree/master/src/lib/tunes
- What is the tune format?
- How are they mapped to events?
- Who can create and use this service?
- What else would you like to know ....

This originates from https://github.com/PX4/Firmware/pull/9538

@MaEtUgR - would you be able to create first draft of this? Or bullet point the answers to the above here and I'll create something!

### PR DESCRIPTION
**Pull request by hamishwillee**
This fixes #528 . It is supposed to explain:
- [x] Tune architecture (lib, definitions, control)
- [x] Where default tunes are defined
- [x] How you can can create your own
- [x] How to replace existing
- [ ] How to add new in code.

@MaEtUgR FYI. This doesn't do the last one, because I don't have time to look into it yet. Also no non-Windows alternatives provided. 
Is it worth adding as-is and extending?

### CODE REVIEW

**Comment by hamishwillee** [comment]
@MaEtUgR as above - is it worth adding as-is and extending?

**Comment by hamishwillee** [comment]
@MaEtUgR What do you think. Merge for now?

**Comment by hamishwillee** [comment]
@MaEtUgR Merging - leaving the TBD because it does still need to be done.

File: en/advanced/system_tunes.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## Testing Tunes
+
+When you're ready to try it out a new tune on PX4, use the [tune_control](../middleware/modules_system.md#tunecontrol) library.
+For example, to test the tune we "created" above you would enter the following command on a console or shell (e.g. the [MAVLink Shell](../debug/system_console.md#mavlink_shell)):
+```sh
+tune_control play -m "150L1O3DL16CL32<B>C<AEL16A"
+```
+
+> **Note** Out of the box, the tune_control is only present on real hardware (not the simulator).
```
Comment (hamishwillee) [inline en/advanced/system_tunes.md:63]: @julianoes FYI, this is still true, but I'm pretty sure I saw a PR in progress to enable support on SITL.

**Comment by MaEtUgR** [comment]
@hamishwillee Sorry for letting you down on this 😬 It went under the radar. Let me check on the guide now.

**Comment by hamishwillee** [comment]
No worries @MaEtUgR - once WE get to a certain level of busyness then we just end up choosing who we disappoint. So its me too.

**Comment by MaEtUgR** [comment]
To customize the tunes for certain events e.g. startup the preferred way is to create a copy of the definition file: https://github.com/PX4/Firmware/blob/master/src/lib/tunes/tune_definition.desc, switch any tunes you like to customize and adjusting the filename in the includes of https://github.com/PX4/Firmware/blob/master/src/lib/tunes/default_tunes.cpp.

Or did you mean something else with the title https://dev.px4.io/master/en/advanced/system_tunes.html#adding-a-new-tune ?

**Pull request merged**